APE24.rst

```diff
@@ -0,0 +1,443 @@
+* customizing the editorial bot to add community-specific tags.
+
+Is open review a deal breaker?
+------------------------------
+
+We do not think so! In the pre-APE 24 process, the reviewer was anonymous by default
+because we modeled the process after traditional astronomical journals.
+Even then, the reviewer had the option to reveal themselves if they wanted to.
+Given that the Astropy ecosystem is open-source anyway, it makes sense for us
+to move away from the journal-style tradition and embrace a more open process that
+pyOpenSci uses.
+
+Rejecting packages
+------------------
+
+If a package is "rejected," it is likely not in technical or domain scope.
+The rejection will come from pyOpenSci Editor-in-Chief or the editor;
+it does *not* have to come from the Astropy community directly and will be objective.
+Rejection can also happen if the maintainer of the package under review
+is unwilling to ensure the package can be used and maintained.
+While in practice, rejection is rare, usually the Editor-in-Chief would be
+the one doing the rejection.
+
+A package may still be listed under pyOpenSci (and JOSS) if it
+passes the general criteria but fails the Astropy-specific ones;
+e.g., a very nice Python package that has little to do with the Astropy ecosystem.
+
+"Re-review" of packages
+-----------------------
+
+**Note: This does not exist yet and probably will not happen before
+this APE is accepted. As of July 2023, pyOpenSci had a somewhat
+manual process to check in on an accepted package.**
+
+One core goal of pyOpenSci is to support scientific Python packages that are
+maintained over time. Due to the lack of resources, it does not do a *full*
+re-review of packages (involving Editor-in-Chief, editor, and 2 reviewers).
+However, it has plans to set up automated checks that track the "health"
+and maintenance level of a package over time, with the goal of identifying
+packages that have become "orphans" (i.e, unmaintained). Once a package is
+identified as needing additional maintenance, it will be flagged for the community.
+
+"Health" might include frequency of commits, releases, CI status, etc.
+Some, if not most, of this data will be collected using
+`devstats <https://github.com/scientific-python/devstats>`_ and
+`repo-review <https://github.com/scientific-python/repo-review>`_ from
+`Scientific Python`_. This information will be displayed on public dashboards.
+
+However, automation can only go so far; for instance, if automation
+is deployed in GitHub Actions, then any package hosted outside of GitHub
+would be excluded from these checks. In that case, manual intervention
+might still be necessary.
+
+If the package is no longer maintained, one of the following can happen:
+
+1. If it is a widely used package and the maintainer wants to see it live on,
+   they will try to help the maintainer to find a new maintainer team;
+2. Otherwise, they will gracefully sunset it from the list of maintained/accepted
+   packages.
```

Comment:
```suggestion
1. If it is a widely used package and the maintainer wants to see it live on,
   pyOpenSci will try to help the maintainer build a new maintainer team;
2. Otherwise, pyOpenSci will gracefully sunset it from the list of maintained/accepted
   packages.
```